src/app/hikes/hikes-list/hikes-list.component.ts

```diff
@@ -15,7 +15,7 @@ export class HikesListComponent implements OnInit {
   constructor(
     private router: Router,
     private hikesService: HikesService,
-  ){ }
+  ) { }
```

Comment:
We should point out that Angular CLI doesn't give you an ESLint settings file, but one should add that immediately to any project.

Reply:
Yeah, ESLint + turning on the `formatOnSave` preference in your editor should be standard IMO.